doc/src/manual/arrays.md

```diff
@@ -230,13 +230,11 @@ The general syntax for indexing into an n-dimensional array A is:
 X = A[I_1, I_2, ..., I_n]
 ```
 
-where each `I_k` may be:
-
-1. A scalar integer
-2. A `Range` of the form `a:b`, or `a:b:c`
-3. A `:` or `Colon()` to select entire dimensions
-4. An arbitrary integer array, including the empty array `[]`
-5. A boolean array to select a vector of elements at its `true` indices
+where each `I_k` may be a scalar integer, an array of integers, or any other
+[supported index](@ref man-supported-index-types). This includes
+[`Colon`](@ref) (`:`) to select all indices within the entire dimension,
```

Comment:
Yup! It links to http://docs.julialang.org/en/latest/stdlib/arrays.html#Base.Colon